### ISSUE TITLE
Work time

### ISSUE THREAD

**Issue by vinaooo** [akaMrNagar/Mindful#117]
**Describe the solution you'd like**

Something like BEDTIME but related to work, maybe study, school, etc... we can put a time to start and time to finish, lunch time, specific apps and websites to block

**Comment by akaMrNagar** [comment]
Seems like you want to schedule or use app in certain period. You can do this in the latest beta release or preview release on GitHub. If you have any doubts do let me know.

**Comment by vinaooo** [comment]
I installed beta version from Github

The feature you mean is that inside Statistics (restriction groups)? If yes I think it should be in Wellbeing, not? Statistics seems to be something like analysis not configurations

I also noticed that accessibility settings are now unavailable to Mindful

**Comment by akaMrNagar** [comment]
Yes you can limit specific app or group of apps by specifying active period by going to app dashboard for specific app or by restriction groups for group of apps.


You can click on the help button in the accessibility permission warning in well-being tab it will redirect you to FAQs where you can find the reason and guide to enable it.
